### PR TITLE
New package: AmeyaP.UTAU version 0.4.18

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/132335)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification. It has the write permission of this account's fork so that it can resolve the issue directly.

>[!important]
>Please recheck these Pull Request carefully before merging. If it is a incorrect-removing Pull Request and the URLs don't have any issue while manual checking, please close this Pull Request directly(It's best to let Exorcism0666 know).

良い一日をお過ごしください！

File: manifests/a/AmeyaP/UTAU/0.4.18/AmeyaP.UTAU.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: AmeyaP.UTAU
+PackageVersion: 0.4.18
```
Comment (Trenly) [inline manifests/a/AmeyaP/UTAU/0.4.18/AmeyaP.UTAU.installer.yaml:5]:
```suggestion
PackageVersion: 1.1.18
```
This should match control panel. If it does not, `AppsAndFeaturesEntries` are required in the manifest

File: manifests/a/AmeyaP/UTAU/0.4.18/AmeyaP.UTAU.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: AmeyaP.UTAU
+PackageVersion: 0.4.18
+InstallerType: zip
+NestedInstallerType: exe
+NestedInstallerFiles:
+- RelativeFilePath: utau0418inst.exe
+InstallerSwitches:
+  Custom: /norestart
```
Comment (Trenly) [inline manifests/a/AmeyaP/UTAU/0.4.18/AmeyaP.UTAU.installer.yaml:11]: This is causing a command line error when trying to install the program. You will need to fix the installer switches so it installs unattended

File: manifests/a/AmeyaP/UTAU/0.4.18/AmeyaP.UTAU.locale.ja-JP.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: AmeyaP.UTAU
+PackageVersion: 0.4.18
```
Comment (Trenly) [inline manifests/a/AmeyaP/UTAU/0.4.18/AmeyaP.UTAU.locale.ja-JP.yaml:5]:
```suggestion
PackageVersion: 1.1.18
```

File: manifests/a/AmeyaP/UTAU/0.4.18/AmeyaP.UTAU.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: AmeyaP.UTAU
+PackageVersion: 0.4.18
```
Comment (Trenly) [inline manifests/a/AmeyaP/UTAU/0.4.18/AmeyaP.UTAU.yaml:5]:
```suggestion
PackageVersion: 1.1.18
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by Exorcism0666** [comment]
:^)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-01-02T16:09:03.1728223Z ##[error] Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\a\AmeyaP\UTAU\0.4.18', expected 'manifests\a\AmeyaP\UTAU\1.1.18'

(Automated response - build 536.)

**Comment by stephengillie** [comment]
Hi @Exorcism0666,

Please update the file paths to match the new version.

**Pull request closed without merging**